completion/available/gulp.completion.bash

```diff
@@ -1,50 +1,5 @@
-#!/bin/bash
-# Borrowed from grunt-cli
-# http://gruntjs.com/
-#
-# Copyright jQuery Foundation and other contributors, https://jquery.org/
+# shellcheck shell=bash
 
-# This software consists of voluntary contributions made by many
-# individuals. For exact contribution history, see the revision history
-# available at https://github.com/gruntjs/grunt .
-
-# The following license applies to all parts of this software except as
-# documented below:
-
-# ====
-
-# Permission is hereby granted, free of charge, to any person obtaining
-# a copy of this software and associated documentation files (the
-# "Software"), to deal in the Software without restriction, including
-# without limitation the rights to use, copy, modify, merge, publish,
-# distribute, sublicense, and/or sell copies of the Software, and to
-# permit persons to whom the Software is furnished to do so, subject to
-# the following conditions:
-
-# The above copyright notice and this permission notice shall be
-# included in all copies or substantial portions of the Software.
-
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-# Usage:
-#
-# To enable bash <tab> completion for gulp, add the following line (minus the
-# leading #, which is the bash comment character) to your ~/.bashrc file:
-#
-# eval "$(gulp --completion=bash)"
-# Enable bash autocompletion.
-function _gulp_completions() {
-# The currently-being-completed word.
-local cur="${COMP_WORDS[COMP_CWORD]}"
-#Grab tasks
-local compls=$(gulp --tasks-simple)
-# Tell complete what stuff to show.
-COMPREPLY=($(compgen -W "$compls" -- "$cur"))
-}
-complete -o default -F _gulp_completions gulp
+if _command_exists gulp; then
+	eval "$(gulp --completion=bash)"
```

Comment:
this also does not always work sadly

Reply:
@NoahGorny It is working

```
 2022-05-20 08:00:37 ⌚  36f05b99f861 in /my-project
○ → git clone --branch improve/gulp-completion  https://github.com/tbhaxor/bash-it.git ~/.bash_it -q

 2022-05-20 08:00:45 ⌚  36f05b99f861 in /my-project
○ → ~/.bash_it/install.sh 

 2022-05-20 08:00:52 ⌚  36f05b99f861 in /my-project
○ → cat ~/.bash_it/completion/available/gulp.completion.bash 
# shellcheck shell=bash

if _command_exists gulp; then
        eval "$(gulp --completion=bash)"
fi

 2022-05-20 08:01:07 ⌚  36f05b99f861 in /my-project
○ → bashit enable completion gulp
gulp enabled with priority 350.

 2022-05-20 08:01:15 ⌚  36f05b99f861 in /my-project
○ → bashit restart 

 2022-05-20 08:01:21 ⌚  36f05b99f861 in /my-project
○ → gulp 
default  help     todo     

 2022-05-20 08:01:21 ⌚  36f05b99f861 in /my-project
○ → gulp 
default  help     todo     

 2022-05-20 08:01:21 ⌚  36f05b99f861 in /my-project
○ → gulp 
[08:01:40] Using gulpfile /my-project/gulpfile.js
[08:01:40] Starting 'default'...
[08:01:40] Finished 'default' after 2.39 ms
```

I have tested this in `GNU bash, version 5.1.16(1)-release (x86_64-alpine-linux-musl)`